pom.xml

```diff
@@ -23,12 +23,11 @@
         <maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <ch-kafka.version>1.4.5</ch-kafka.version>
-        <structured-logging.version>1.9.11</structured-logging.version>
+        <structured-logging.version>1.9.12</structured-logging.version>
```

Comment:
The latest version of structured logging is 1.9.14. Might as well update to that